src/test/resources/json/WikiTest/testWikiMapping/files/57d87dd6da3e2401ed91bbfe/versions/index.json

```diff
@@ -0,0 +1,26 @@
+{
+    "data": [
+        {
+            "links": {
+                "self": "http://localhost:8000/v2/files/57d87dd6da3e2401ed91bbfe/versions/1/",
+                "html": "https://test.osf.io/ng9em/files/osfstorage/57d87dd6da3e2401ed91bbfe?revision=1"
```

Comment:
Sounds good.  I had just done a global search/replace on URLs I knew to be problematic.